### ISSUE TITLE
ENTRYPOINT ["bin/sh"]

### ISSUE THREAD

**Issue by pineking** [jbender/docker-kaldi#2]
the last line in base/Dockerfile 
ENTRYPOINT ["bin/sh"]

should it be ENTRYPOINT ["/bin/sh"]?

**Comment by jbender** [comment]
Yes, you're correct, and I'd welcome a PR to that effect. It's been a couple years since I last touched this so I'm sure some things have changed since it was originally working.
